stream-chat-android-ui-components/src/main/res/layout/stream_attachment_gallery.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/vp_attachmentGallery"
+        android:layout_width="match_parent"
```

Comment:
Not a big deal, but I had some subtle bugs when using `match_parent`. `0dp` is better.